raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -467,6 +467,7 @@ def test_join_invalid_discovery(
 @pytest.mark.flaky(max_runs=5)
```

Comment:
Do we want to keep these when we add a skip marker?

Reply:
The flaky marker is new to me so I did not mess with it. But due to your comment I looked it up and I see what it does. So perhaps it should go too. Let me handle that.